test/SlowTests/Issues/RavenDB-23136.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Utils;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Smuggler;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23136 : RavenTestBase
+    {
+        public RavenDB_23136(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenTheory(RavenTestCategory.Smuggler | RavenTestCategory.Revisions)]
+        [RavenData(Data = new object[] { false })]
+        [RavenData(Data = new object[] { true })]
+        public async Task Import_Should_Create_Revisions_When_Configuration_Is_On(Options _, bool withRevisionsConfig)
```

Comment:
Do we want to have this test for sharded databases as well?
if yes - pass this options properly.
if not - remove it.